services/web/server/tests/unit/with_dbs/04/workspaces/test_workspaces.py

```diff
@@ -151,3 +148,4 @@ async def test_project_workspace_movement_full_workflow(
     assert client.app
 
     # NOTE: MD: not yet implemented
+    # SEE https://github.com/ITISFoundation/osparc-simcore/issues/6778
```

Comment:
Thanks, I will take care of this. The test name is misleading. The Folder to Workspace feature is not yet implemented (probably what I meant)